docs/reference/8-admin-app/technical-articles/configuration-settings-v2x.md

```diff
@@ -0,0 +1,44 @@
+| **DefaultOdsInstance** | Sets the default ODS instance name to be used in case of SharedInstance mode. | Any string.   Default value: "EdFi ODS" |
+| **ProductionApiUrl** | Points to the Ed-Fi ODS / API server url. | Any string (must be a valid server url) |
+| **SecurityMetadataCacheTimeoutMinutes** | Sets the amount of time the security metadata from the EdFi\_Security database is cached. E.g., if it is set to 10 mins, the claim set changes will reflect in the API at least after 10 mins without needing to recycle the API process. **This setting must match the same setting on the ODS/API appsettings.json.** | A positive integer representing the number of minutes. Default value: 10 |
+| **ApiStartupType** | Sets the component settings for the API. **This setting must match the ODS/API settings.** See the [following section](https://edfi.atlassian.net/wiki/pages/viewpage.action?pageId=20480170#PlatformDevGuideExtensibility&Customization-DbPartition) on Developers' Guide for more information on this setting. | "SharedInstance", "YearSpecific", "DistrictSpecific".  Default value: "SharedInstance" |
+| **LocalEducationAgencyTypeValue** | Sets the name for the 'Local Education Agency' education organization type. | Any string. Default value: "Local Education Agency" |
+| **SchoolTypeValue** | Sets the name for the 'School' education organization type. | Any string. Default value: "School" |
+| **BulkUploadHashCache** | Sets the folder path to the Bulk Upload hash cache. | Any string (must be an existing folder path). Default value: "C:\\\\ProgramData\\\\Ed-Fi-ODS-AdminApp\\\\BulkUploadHashCache" |
+| ****OptionalEntropy  ****> [!CAUTION]> Admin App 2.2x onwards, this appsetting is deprecated. The only encryption protocol supported is AES. This appsetting is, therefore, no longer needed. | Sets the optional entropy value for DPAPI encryption and decryption calls. Entropy is an additional key specific to the application that is protecting the data. If not specified, it allows other applications running on the same machine to decrypt the encrypted data. If specified, it allows for the protection of data from other system applications. | Any string |
+| **EncryptionProtocol**> [!CAUTION]> Admin App 2.2x onwards, this appsetting is deprecated. The only encryption protocol supported is AES. This appsetting is, therefore, no longer needed. | Sets the encryption protocol to be used. Data Protection API (DPAPI) is the default encryption protocol. AES encryption is used for docker deployments. | "DPAPI", "AES". Default value: "DPAPI" |
+| **Cloud ODS Specific App Settings - Set by Cloud ODS Deployment and not recommended to set manually.** |     |     |
+| **IdaAADInstance** | Sets the base URL of the authorization server (this is always [https://login.microsoftonline.com](https://login.microsoftonline.com)) |     |
```

Comment:
We probably don't want this to be a link. Recommendation: change to 

~~~
`https://login.microsoftonline.com`
~~~

That way it is just a simple string and not a hyperlink.